core/src/main/resources/lib/layout/breadcrumbs.css

```diff
@@ -89,38 +89,52 @@
     display: inline-block;
     display: inline-flex;
     align-items: center;
-    background-image: url(menu_right_arrow2.png);
-    background-position: center center;
-    background-repeat: no-repeat;
+    justify-content: center;
+}
 
+#breadcrumbs LI.children:after, #breadcrumbs LI.separator:after {
+    /* Right arrow */
+    content: "";
+    border-left: 0.25em solid #bcbcbc;
+    border-top: 0.25em solid transparent;
+    border-bottom: 0.25em solid transparent;
+    border-right: 0;
 }
 
 #breadcrumbs LI.children {
     cursor: pointer;
 }
 
-#breadcrumbs LI.children:hover {
-    background-image: url(menu_right_arrow.png);
+#breadcrumbs LI.children:hover:after {
+    border-left-color: #4d545d;
```

Comment:
Will need update after https://github.com/jenkinsci/jenkins/pull/4752 is merged so that we do not hardcode colors

Reply:
And I think the variable should be `--text-color`.